lib/modules/datasource/gitea-tags/index.ts

```diff
@@ -16,6 +16,13 @@ export class GiteaTagsDatasource extends Datasource {
 
   private static readonly cacheNamespace: PackageCacheNamespace = `datasource-${GiteaTagsDatasource.id}`;
 
+  override readonly releaseTimestampSupport = true;
+  override readonly releaseTimestampNote =
+    'The release timestamp is determined from the `created` field in the results.';
+  override readonly sourceUrlSupport = 'package';
+  override readonly sourceUrlNote =
+    'The source url determined by using the `packageName` and `registryUrl`.';
```

Comment:
```suggestion
    'The source URL is determined by using the `packageName` and `registryUrl`.';
```

This seems to be the correct pattern?